code/src/java/pcgen/core/PlayerCharacter.java

```diff
@@ -405,15 +397,15 @@
 	private MasterFacet masterFacet = FacetLibrary.getFacet(MasterFacet.class);
 	private AutoEquipmentListFacet autoListEquipmentFacet = FacetLibrary.getFacet(AutoEquipmentListFacet.class);
 	private FollowerFacet followerFacet = FacetLibrary.getFacet(FollowerFacet.class);
-	
+
```

Comment:
also, I plan on making more changes to PlayerCharacter so I'd prefer to just set my IDE correctly for pcgen and let it make the fixes